src/lib/request.js

```diff
@@ -30,6 +30,7 @@ export default function (relativeUrl, opts, callback) {
     const headers = {
         'stencil-config': options.requestOptions.config ? JSON.stringify(options.requestOptions.config) : '{}',
         'stencil-options': '{}',
+        'x-xsrf-token': window.BCData.csrf_token ? window.BCData.csrf_token : '',
```

Comment:
You should guard this because we don't always set `window.BCData`. So I would write `window.BCData && window.BCData.csrf_token`

Reply:
I would prefer that this data come in from `opts`, but I understand that would be a larger change, requiring updates to all calls to pass in this data, so I'm ok with this approach for something like this that is a platform-level concern.